### PR TITLE
Avijit/18.1-rc0

### PR DESCRIPTION
**Pull request by avijit-nervana**
This is patch release for changes since 18.0

### CODE REVIEW

**Comment by avijit-nervana** [comment]
@googlebot I consent.

**Comment by XWangIntel** [comment]
@googlebot I consent

File: README.md [hunk 1]
```diff
@@ -127,8 +127,8 @@ Verify that `ngraph-bridge` installed correctly:
     This will produce something like this:
 
         TensorFlow version:  <1.14.0>
-        nGraph bridge version: <b'0.18.0'>
-        nGraph version used for this build: b'0.25.0+005c118'
+        nGraph bridge version: <version number>
+        nGraph version used for this build: b'Version nuymber'
```
Comment (shresthamalik) [inline README.md]:
```suggestion
        nGraph version used for this build: b'Version number'
```

**Comment by shresthamalik** [comment]
Probably needs a merge with master to get cpu-variables pipeline running. Not seeing an option to update the branch

**Comment by sindhu-nervana** [comment]
@shresthamalik This PR is branched off r0.18

**Comment by shresthamalik** [comment]
So the only way to get the pipeline running on this PR is by cherrypicking the commit

**Pull request merged**